src/routers/sra_router.ts

```diff
@@ -1,10 +1,23 @@
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
 import * as express from 'express';
 import * as asyncHandler from 'express-async-handler';
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
 
+import { SRA_PATH } from '../constants';
 import { SRAHandlers } from '../handlers/sra_handlers';
+import { errorHandler } from '../middleware/error_handling';
 import { OrderBookService } from '../services/orderbook_service';
 
-export const createSRARouter = (orderBook: OrderBookService): express.Router => {
+export const configureSRAHttpRouter = (app: core.Express, orderBook: OrderBookService): void => {
+    app.use(cors());
```

Comment:
Yeah, this was happening before the refactor too. Was just copying code snippets around. On second thought, all of this seems unnecessary. I'm going to remove all the `configure*` methods.

Reply:
Not 100% what you mean but this architecture is there partially so that you can easily delete code in your own fork of the API (if you only want SRA for example).